### DISCUSSION TITLE
Support Google Assistant API

### DISCUSSION THREAD

**Discussion by Wireheadbe** [andrei-tatar/node-red-contrib-smartnora#138] [Ideas]
The Google Assistant API has support for a new feature called "text_query"
https://developers.google.com/assistant/sdk/reference/rpc/google.assistant.embedded.v1alpha2#assistconfig

This would make an awesome node for SmartNora home automation! Previously, it was impossible to automate things like "Ok Google, Play "Show" on "Chromecast" via "Disney Plus" " without crazy workarounds like playing back audio and capturing it on another smart device, or using phones with ADB debug and what not.

**Comment by andrei-tatar** [comment]
As far as I understood, that SDK is meant for building your own *Google Home* device.

> The Google Assistant SDK lets you add voice control, natural language understanding and Google’s smarts to your ideas. Your project captures an utterance (a spoken audio request, such as What's on my calendar?), sends it to the Google Assistant, and receives a spoken audio response in addition to the raw text of the utterance.
